lib/common.sh

```diff
@@ -1,8 +1,23 @@
 #!/usr/bin/env bash
 
-CONFIG_FILE="$HOME/.stackrox/workflow-config.json"
+# Use CONFIG_HOME if it exists. Otherwise follow XDG base directory specification.
+CONFIG_HOME="${HOME}/.stackrox/workflow-config.json"
+if [ -f "${CONFIG_FILE}" ]; then
+	CONFIG_FILE=$CONFIG_HOME
+else
+	XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-${HOME}/.config}"
```

Comment:
Syntax seems wrong? (I think the `}` should be before the `.config`)

Reply:
Sorry I don't get what you mean. I think the syntax is correct - if `XDG_CONFIG_HOME` exists then do nothing, if it does exist then default with `${HOME}/.config` per XDG spec.

Reply:
But shouldn't it be like this? 🤔 
```suggestion
	XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-${HOME}}/.config"
```

Reply:
Ohh I see. You don't want to unconditionally append `/.config`. Nvm, I misunderstood, sorry.